angular/src/lib/modal/modal-header.component.ts

```diff
@@ -20,7 +20,6 @@ import { ModalRef } from './modal-ref';
     *ngIf="showCloseButton"
     class="md-close md-modal__close"
     (click)="close()"
-    (keydown)="handleKeydown($event)"
```

Comment:
so .. can you still use the keyboard to close the modal?

Reply:
Yes, the browser automatically triggers a click event when you press the Enter or Space keys. This will call `close()` and close the modal.

Reply:
essentially this block of code will make sure that `ESC` key will close the modal https://github.com/momentum-design/momentum-ui/blob/ab519026a8310b1452132eac1bdfbbedff8f81d8/angular/src/lib/modal/modal-header.component.ts#L45

Reply:
@mohammedzamakhan That's correct. We don't have any issue with that though. The listener is on `document`, so the ESC key will always close the modal.

The `(keydown)` event **on** the button is problematic because, when the focus is on the button, pressing *any* key causes the modal to be closed. For example: if you open a modal and just press Tab repeatedly, you will eventually close the modal (the focus will eventually be on the button, and then, one more Tab key press will close the modal).